lib/kino/data_table.ex

```diff
@@ -83,7 +92,8 @@ defmodule Kino.DataTable do
   """
   def update(kino, tabular, opts \\ []) do
     {data_rows, data_columns, count, inspected} = prepare_data(tabular, opts)
-    Kino.Table.update(kino, {data_rows, data_columns, count, inspected})
+    formatter = Keyword.get(opts, :formatter, &value_to_string/2)
+    Kino.Table.update(kino, {data_rows, data_columns, count, inspected, formatter})
```

Comment:
`update` is rather designed for updating data and I wouldn't expect formatter to change after the table is created. So let's support it only in `new` :)